### ISSUE TITLE
Generating mixture model data

### ISSUE THREAD

**Issue by pnewstein** [chenyenchung/scMarco#1]
Thanks for making this tool! I am interested in using this on my own single cell dataset. However, I haven't been able to calculate gene expression probability for identified clusters.

The documentation begins assuming that you have previously calculated modeled expression probability. In the paper, it looks like the [code ](https://github.com/chenyenchung/mixture_modeling/blob/master/src/R/analyzeOpticLobeExpr.R) from [Davis et al., 2020](https://elifesciences.org/articles/50901) was used to get these modeled probabilities. Is that correct?

I have had trouble using that code on my own data, since I don't know how to format my data. Can you provide an example on how to use the code in analyzeOpticLobeExpr.R? 

Thank you,

Peter

**Comment by chenyenchung** [comment]
Hi Peter,

I am really sorry for the late reply. Not sure why I did not get the notification in my inbox.

We realized that the original work from Davis and colleagues could be a bit hard to follow, so we wrapped our fork in a Snakemake pipeline in [this repository](https://github.com/chenyenchung/mixture_modeling_smk). If you still need it, please let us know if you have any questions with it.

Best,
Yen
